dev-ml/camlp5/camlp5-7.10.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_P=${P%_p*}
+DESCRIPTION="A preprocessor-pretty-printer of ocaml"
+HOMEPAGE="https://camlp5.github.io/"
+SRC_URI="https://github.com/camlp5/camlp5/archive/rel$(ver_rs 1- '').tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE="doc +ocamlopt"
+DEPEND="
+	<=dev-lang/ocaml-4.6.0:=[ocamlopt?]
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}-rel$(ver_rs 1- '')"
+
+src_prepare() {
+	eapply_user
+}
```

Comment:
This should be default behaviour and not needed here.

Reply:
Yes.